File: 2310817120009_AlysaArmelia_PemrogramanI_Modul4/Python/PRAK405-2310817120009-AlysaArmelia.py
m, n = map(int,input().split())

perkalian_kelipatan = 0

nilai = 0

for k in range(1, m+1):
    keseluruhan = 0

    for l in range(k, 0, -1):
        perkalian_kelipatan = l * n
        print(f"({l} * {n})", end="")

        keseluruhan += perkalian_kelipatan

        if l > 1:
            print(" + ", end="")

    print(f" = {keseluruhan}")
    nilai += keseluruhan

print(nilai)

File: 2310817120009_AlysaArmelia_PemrogramanI_Modul4/Bahasa C/PRAK405-2310817120009-AlysaArmelia.c
#include <stdio.h>

int main()
{
    int m;
    int n;
    int perkalian_kelipatan;

    scanf("%d %d", &m, &n);

    int nilai= 0;

    for (int k = 1; k <= m; k++){

        int keseluruhan = 0;
        for (int l = k; l >= 1; l--)
        {
            perkalian_kelipatan = l * n;
            printf("(%d * %d)", l, n);

            keseluruhan += perkalian_kelipatan;

            if(l > 1){
                printf(" + ");
            }
        }
        printf(" = %d \n", keseluruhan);
        nilai += keseluruhan;
    }
    printf("%d \n", nilai);
return 0;
}
